source/components/organisms/SummaryList/SummaryList.tsx

```diff
@@ -128,9 +134,23 @@ const SummaryList: React.FC<Props> = ({
   help,
   editable,
 }) => {
-  // For development: show an error popup when something is configured wrong.
-  const showNotification = useNotification();
+  const [isModified, setModified] = useState(false);
 
+  const sortAnswers = (listItems: SummaryListItem[]) => {
```

Comment:
Rename `listItems` so it doesn't collide with same variable one scope up.

Reply:
Done